From f35c4949127ed15c46771fcc10b89db4c4439999 Mon Sep 17 00:00:00 2001
From: Adithya R <gh0strider.2k18.reborn@gmail.com>
Date: Thu, 26 May 2022 23:43:56 +0530
Subject: [PATCH 06/12] SystemUI: Fix QS clock overlapping on UI mode change

Calling updateAnimators() in a separate runnable seems to
cause QSB header elements to overlap each other when switching
between dark and light mode. This change was introduced in
https://github.com/LineageOS/android_frameworks_base/commit/0558e1211604b3b6f465a394832bbf8fa06d1dfe and seems unnecessary so lets revert it.

Change-Id: I14e60b608f2aac373e71fec0a6ab7d8e787d2457
---
 .../src/com/android/systemui/qs/QuickStatusBarHeader.java       | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/src/com/android/systemui/qs/QuickStatusBarHeader.java b/packages/SystemUI/src/com/android/systemui/qs/QuickStatusBarHeader.java
index 982306cd8df9..baf93f31d5a5 100644
--- a/packages/SystemUI/src/com/android/systemui/qs/QuickStatusBarHeader.java
+++ b/packages/SystemUI/src/com/android/systemui/qs/QuickStatusBarHeader.java
@@ -194,7 +194,7 @@ public class QuickStatusBarHeader extends FrameLayout {
         super.onMeasure(widthMeasureSpec, heightMeasureSpec);
         if (mDatePrivacyView.getMeasuredHeight() != mTopViewMeasureHeight) {
             mTopViewMeasureHeight = mDatePrivacyView.getMeasuredHeight();
-            post(this::updateAnimators);
+            updateAnimators();
         }
     }
 
-- 
2.32.0

